de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,111 @@
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=.nagstamon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --filesystem=xdg-run/keyring
+  - --filesystem=home
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: libsecret
+    buildsystem: meson
+    config-opts:
+      - -Dgcrypt=false
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+      - -Dmanpage=false
+      - -Dvapi=false
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/libsecret/-/archive/0.20.5/libsecret-0.20.5.tar.gz
+        sha256: b33b9542222ea8866f6ff2d31c0ad373877c2277db546ca00cc7fdda9cbab1c3
+  - name: krb5
+    subdir: src
+    config-opts:
+      - --localstatedir=/var/lib
+      - --sbindir=${FLATPAK_DEST}/bin
+      - --disable-static
+      - --disable-rpath
+    cleanup:
+      - /bin
+      - /share/et
+      - /share/examples
+      - /share/man
+    sources:
+      - type: archive
+        url: https://github.com/krb5/krb5/archive/refs/tags/krb5-1.21.2-final.tar.gz
+        sha256: 5827fa39ead0e7f7b8a158799eb5d50b194129e81b378892caedcafe50c5c195
+      - type: shell
+        dest: src
+        commands:
+          - autoreconf -si
+  - name: dbus-run-session
+    buildsystem: autotools
+    cleanup: ['*']
+    sources:
+      - type: archive
+        url: https://gitlab.freedesktop.org/dbus/dbus/-/archive/dbus-1.14.10/dbus-dbus-1.14.10.tar.gz
+        sha256: 5fedcd6c17fbc62646af224e1aab11a334c463e7e7d89163a670ed24e7ab241c
+        x-checker-data:
+          type: anitya
+          project-id: 5356
+          url-template: https://gitlab.freedesktop.org/dbus/dbus/-/archive/dbus-$version/dbus-dbus-$version.tar.gz
+  - name: dbus-python
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
+        sha256: ad67819308618b5069537be237f8e68ca1c7fcc95ee4a121fe6845b1418248f8
+        x-checker-data:
+          type: anitya
+          project-id: 402
+          url-template: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$version.tar.gz
+
+  - modules/python3-cryptography.yml
+  - modules/python3-lxml.yml
+  - modules/python3-beautifulsoup4.yml
+  - modules/python3-keyring.yml
+  - modules/python3-requests.yml
+  - modules/python3-psutil.yml
+  - modules/python3-dateutil.yml
+  - modules/python3-requests-kerberos.yml
+
+  - name: setup
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py bdist --format tar
+      - tar -xvf dist/nagstamon*.tar --strip 2 -C /app
+      - install -D nagstamon.sh /app/bin/nagstamon
+      - mkdir -p /app/share/icons/hicolor/scalable/apps/
+      - sed -i 's/416\.04276/412\.2182/g' /app/share/pixmaps/nagstamon.svg
```

Comment:
This should be fixed upstream, changing this here seems like a hack.

Reply:
or at least not using `sed`. Patches are a better alternative if fix is needed.

Reply:
The next stable release will contain the patched file with exact same x and y lengths.

Reply:
@bbhtt @hfiguiere I created a patchfile but cannot access it while building. Can you provide me a hint?

Reply:
Patch file is now included.